recipes/pytest-ast-back-to-python/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pytest-ast-back-to-python" %}
+{% set version = "1.0.1" %}
+{% set hash = "c459b3ea9b0a41f2cdf6c80b7642db1ee40d74b98b58434749932190b82c2e4e" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0]}}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pytest >=3.0.0
+
+test:
+  imports:
+    - pytest  # must be imported first because of circular imports in pytest_ast_back_to_python
+    - pytest_ast_back_to_python
+
+about:
+  home: https://github.com/tomviner/pytest-ast-back-to-python
+  license: BSD-3
+  license_family: BSD
```

Comment:
note: No license file yet, but an issue has been raised upstream.

xref: https://github.com/tomviner/pytest-ast-back-to-python/issues/5